npm/private/npm_translate_lock_helpers.bzl

```diff
@@ -57,15 +58,38 @@ def _gather_package_content_excludes(keyed_lists, *names):
             v = keyed_lists[name]
             if type(v) == "list":
                 for e in v:
-                    excludes.append(e)
+                    if e == "yarn-autoclean":
+                        # Support string presets within lists
+                        excludes.extend(exclude_package_contents_default)
+                    else:
+                        excludes.append(e)
             elif type(v) == "string":
-                excludes.append(v)
+                # Support string presets
+                if v == "yarn-autoclean":
+                    excludes.extend(exclude_package_contents_default)
+                else:
+                    excludes.append(v)
             else:
                 fail("expected value to be list or string")
 
     # in case the key has not been met even once, we return None, instead of empty list as empty list is a valid value
     if not found and "*" in keyed_lists:
-        excludes = keyed_lists["*"] if type(keyed_lists["*"]) == "list" else [keyed_lists["*"]]
+        v = keyed_lists["*"]
+        if type(v) == "list":
+            for e in v:
+                if e == "yarn-autoclean":
```

Comment:
instead of a magic string, how about `True` means you want the yarn-autoclean default list